init.sql

```diff
@@ -0,0 +1,8 @@
+GRANT ALL PRIVILEGES ON app_db.* TO "app_user"@"%";
+
+USE app_db;
+
+CREATE TABLE counter (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	value int
+);
```

Comment:
![image](https://github.com/user-attachments/assets/1cabbbd6-f366-4951-b3b9-dd952b154548)
The same, add EoF